package.json

```diff
@@ -223,6 +223,17 @@
         ],
         "path": "./snippets/javascript/javascript.json"
       },
+      {
+        "language": [
```

Comment:
Why not specify the language as "jsdoc" so that users could then extend their local snippets to include these? 

Example: 
```lua
require("luasnip").filetype_extend("typescript", { "jsdoc" })
```

My thinking is this would allow users to still use the snippets while not overloading others who may not want them to load in?

Reply:
That makes sense. I've updated to reflect this suggestion

Reply:
Ah groovy, LGTM !